timer/src/components/Timer.js

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
+import { useTimer } from './TimerContext';
+import { useTimerStyles } from './TimerStyles';
+
+export const Timer = (props) => {
+
+  const { timer } = props
+
+  const [elapsedTime, setElapsedTime] = useState(timer.elapsedTime)
+  const [startTime, setStartTime] = useState(timer.startTime)
+  const [isRunning, setIsRunning] = useState(timer.isRunning)
+
+  const { actions } = useTimer()
+  const timerClasses = useTimerStyles()
+
+  let timerInterval;
```

Comment:
read about `useRef` for this kind of variables, this usage is incorrect because in every render it will be erased